### PR TITLE
Add header for build image in new pdc client output

### PR DESCRIPTION
**Pull request by ycheng-aa**
JIRA: 1277

### CODE REVIEW

File: pdc_client/plugins/build_images.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ def register(self):
     def _print_build_image_list(self, build_images, with_md5=False):
         fmt = '{image_id}'
         if with_md5:
-            fmt += ' {md5}'
+            fmt = '{image_id:50}{md5}'
+        print fmt.format(image_id='Image ID', md5='MD5')
+        print
```
Comment (bliuredhat) [inline pdc_client/plugins/build_images.py:44]: why need to add a blank line ?

**Comment by simozhan** [comment]
:+1:

File: pdc_client/plugins/build_images.py (hunk 1 above)
Comment (ycheng-aa) [inline pdc_client/plugins/build_images.py:44]: For me it looks more clear. It also displayed like this way in compose-image-rtt-tests, compose-tree-locations. Not sure if this way is real better. @simozhan what do you think?

Comment (simozhan) [inline pdc_client/plugins/build_images.py:44]: i have the same idea. from the output, it is more clear to add a new line between the results and the title.

**Comment by lubomir** [comment]
Looks good to me.

**Pull request merged**